resolwe/flow/managers/listener/listener.py

```diff
@@ -69,22 +69,155 @@
 logger = logging.getLogger(__name__)
 
 
-class Processor:
-    """Class respresents instance of one connected worker."""
+class RedisCache:
+    """The Redis cache."""
+
+    cached_fields_type = {
+        "id": int,
+        "slug": str,
+        "contributor": int,
+        "contributor_id": int,
+        "status": str,
```

Comment:
This can change.

Reply:
Sure. But can the change be made by something that is not listener? 
The listener uses method for saving changes that also updates the Redis cache.

Reply:
You are right.